modules/psap-configuring-node-feature-discovery.adoc

```diff
@@ -0,0 +1,380 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/psap-node-feature-discovery-operator.adoc
+
+[id="configuring-the-node-feature-discovery_{context}"]
+= Configuring the Node Feature Discovery Operator
+
+[id="configuring-node-feature-discovery-operator-core_{context}"]
+== core
```

Comment:
I am wondering if it is better to number the yaml sections that need to be explained like for eg. in here - https://docs.openshift.com/container-platform/4.7/installing/installing_aws/installing-aws-customizations.html#installation-aws-config-yaml_installing-aws-customizations

Reply:
I discussed this with Eduardo and we can create a follow-up PR in the future to restructure that section.